### PR TITLE
feat: Add default configmaps - Part 5

### PR DESCRIPTION
**Pull request by ishaansehgal99**
**Reason for Change**:
Provide a default configmaps in the helm chart.

**Notes for Reviewers**:
An env variable release namespace is passed in for future use so that the controller will be able to find where the configmap is deployed.

### CODE REVIEW

File: charts/kaito/workspace/templates/clusterrole.yaml [hunk 1]
```diff
@@ -21,6 +21,9 @@ rules:
   - apiGroups: [ "" ]
     resources: [ "pods"]
     verbs: ["get","list","watch","create", "update", "patch" ]
+  - apiGroups: [ "" ]
+    resources: [ "configmaps" ]
+    verbs: [ "get","list","watch","create", "delete" ]
```
Comment (Fei-Guo) [inline charts/kaito/workspace/templates/clusterrole.yaml:26]: Do we really need to "create" or "delete" configmaps in the kaito controller?

File: charts/kaito/workspace/templates/deployment.yaml [hunk 2]
```diff
@@ -43,6 +43,8 @@ spec:
                   fieldPath: metadata.namespace
             - name: PRESET_REGISTRY_NAME
               value: {{ .Values.presetRegistryName }}
+            - name: RELEASE_NAMESPACE
```
Comment (Fei-Guo) [inline charts/kaito/workspace/templates/deployment.yaml:46]: You don't need this. The controller can find the namespace in the Pod's local directory. 

/var/run/secrets/kubernetes.io/serviceaccount/namespace

Comment (ishaansehgal99) [reply charts/kaito/workspace/templates/deployment.yaml:46]: ok cool removed

File: charts/kaito/workspace/templates/clusterrole.yaml (hunk 1 above)
Comment (ishaansehgal99) [reply charts/kaito/workspace/templates/clusterrole.yaml:26]: no removed

**Review by Fei-Guo — APPROVED** [review]

**Pull request merged**